io.qt.QtCreator.json

```diff
@@ -0,0 +1,40 @@
+{
+    "app-id": "io.qt.QtCreator",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "qtcreator",
+    "rename-desktop-file": "org.qt-project.qtcreator.desktop",
```

Comment:
I'd say that if that's the ID used universally across qtcreator codebase, we should stick to it.

Reply:
@barthalion I didn't remember why I had to change it, but now the error popped up  again:
`error: 'org.qt-project.QtCreator' is not a valid application name: Only last name segment can contain -`

What would you like me to do?